### DISCUSSION TITLE
Add 'order' to BusinessRule

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#3223] [Ideas]
Hi All,
How about adding a property 'order' to BusinessRule?
Something similar to tab index in UI. It would be easier for users to relate broken business rules to UI components.

**Comment by rockfordlhotka** [comment]
You are suggesting that a rule be defined (optionally) with a `DisplayOrder` value (or something) which is purely for UI purposes, and isn't related to `Priority`?

**Reply by Chicagoan2016** [reply]
@rockfordlhotka  Yes 😊
